mne/dipole.py

```diff
@@ -269,15 +269,16 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             title=title)
 
     @verbose
-    def to_mni(self, subject, mri_head_t, subjects_dir=None,
+    def to_mni(self, subject, trans, subjects_dir=None,
                verbose=None):
         """Convert dipole location from head coordinate system to MNI coordinates.
 
         Parameters
         ----------
         %(subject)s
-        mri_head_t : instance of Transform
-            MRI<->Head coordinate transformation.
+        trans : str | None
```

Comment:
if you follow the general API we are following with trans, it can be `str | None or instance of Transform`

see how it's done elsewhere

Reply:
is there a way to factorize the docstring using the shared docdict?

Reply:
I think I did if finally :)